model-engine/model_engine_server/common/dtos/llms.py

```diff
@@ -333,6 +335,18 @@ def total_tokens_per_second(self) -> float:
             else 0.0
         )
 
+    @property
+    def inter_token_latency(self) -> Optional[float]:  # Only for streaming requests
+        if (
+            self.time_to_first_token is None
+            or self.num_completion_tokens is None
+            or self.total_duration is None
+        ):
+            return None
+        if self.num_completion_tokens < 2:
+            return 0.0
```

Comment:
should this return None since there were no tokens after the first?

Reply:
yeah good point, made the change